nwb_conversion_tools/schemas/yaml_conversion_specification_schema.json

```diff
@@ -0,0 +1,51 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "yaml_conversion_specification_schema.json",
+  "title": "YAML conversion specification schema",
+  "description": "Schema for a NWB conversion specified by a YAML file.",
+  "version": "0.1.0",
+  "type": "object",
+  "allOf": [{"$ref": "#/$defs/level"}],
+  "required": ["experiments"],
+  "properties": {
+    "experiments": {
+      "type": "object",
+      "additionalProperties": {
+        "type": "object",
+        "allOf": [{"$ref": "#/$defs/level"}],
+        "required": ["sessions"],
+        "properties": {
+          "sessions": {
+            "type": "array",
+            "items": {
+              "title": "Single-session specification",
+              "type": "object",
+              "required": ["nwbfile_name", "source_data"],
```

Comment:
should nwbfile_name be required? Can we just default to the DANDI naming scheme?

Reply:
Oh, sorry. We removed that previously but I must have restored it to a point slightly before, or forgotten to push it originally. Fixed now.